uptick/pools.py

```diff
@@ -0,0 +1,350 @@
+import click
+from bitshares.account import Account
+from bitshares.amount import Amount
+from bitshares.asset import Asset
+from bitshares.price import Price
+from .decorators import online, unlock
+from .main import main, config
+from .ui import print_tx, format_tx, print_table, print_message
+
+
+@main.group()
+def pool():
+    """ Liquidity pool commands
+    """
+    pass
+
+
+@pool.command()
+@click.argument("asset_a")
+@click.argument("asset_b")
+@click.argument("share_asset")
+@click.argument("taker_fee", type=float)
+@click.argument("withdrawal_fee", type=float)
+@click.option(
+    "--account", help="Active account (else use wallet default). " +
+    "This account must be owner of the SHARE_ASSET."
+)
+@click.pass_context
+@online
+@unlock
+def create(ctx, asset_a, asset_b, share_asset, taker_fee, withdrawal_fee, account):
+    """ Create a new Liquidity Pool.
+
+    ASSET_A, ASSET_B: These are the assets to be traded in the
+    pool. Can be symbols or ids. Note that ASSET_A should be the one
+    that has the lowest-numbered asset id.
+
+    SHARE_ASSET: This is the asset that represents a share in the pool
+    for pool stakers. This asset must be a UIA and have zero supply in
+    order to bind it to the pool at creation, and you must be the
+    owner of this asset.
+
+    TAKER_FEE: This is the fee percent taken by the pool on exchanges
+    made via the pool.  Expressed as a percent from 0.00 to 100.00,
+    where "1%" is expressed as 1.00.
+
+    WITHDRAWAL_FEE: This is the fee percent taken by the pool on withdrawal
+    of stake from the pool.  Expressed as a percent from 0.00 to 100.00,
+    where "1%" is expressed as 1.00.
+
+    """
+    ctx.blockchain.blocking = True
+    tx = ctx.blockchain.create_liquidity_pool(
+        asset_a,
+        asset_b,
+        share_asset,
+        taker_fee,
+        withdrawal_fee,
+        account=account,
+    )
+    tx.pop("trx", None)
```

Comment:
Would it make sense to add a method to `uptick.cli` to simpliy this. Something like `print_tx_and_results(tx)`?

Reply:
Thought about this a bit, and yes.  Seems worth a separate effort and generalizing it to fit and be used by other commands as well.  For now what's here is meant as a "it gets the job done" solution, but I agree it would be useful to add something like this to `ui.py`.

It would actually be cool if it did some smart formatting of the results returned, with interpretation.  E.g. identifying by name created or modified objects, accounts, etc.  Especially when the user creates a proposal.  E.g.: You created proposal 1.x.y for account XXX to do YYY."